src/main/java/uk/ac/cam/cl/dtg/segue/api/monitors/InMemoryMisuseMonitor.java

```diff
@@ -157,7 +160,23 @@ public void resetMisuseCount(final String agentIdentifier, final String eventLab
         
         existingHistory.remove(eventLabel);
     }
-    
+
+    @Override
+    public List<MisuseStatisticDTO> getMisuseStatistics(final String agentIdentifier) {
```

Comment:
See my comment above, I am unsure this is that useful given how varied the agent identifiers are.